ibis/backends/duckdb/__init__.py

```diff
@@ -6,38 +6,43 @@
 import contextlib
 import os
 import warnings
+from operator import itemgetter
 from pathlib import Path
 from typing import TYPE_CHECKING, Any
+from urllib.parse import parse_qs, urlparse
 
 import duckdb
+import pandas as pd
 import pyarrow as pa
 import pyarrow_hotfix  # noqa: F401
-import sqlalchemy as sa
 import sqlglot as sg
-import toolz
+import sqlglot.expressions as sge
 
+import ibis
 import ibis.common.exceptions as exc
 import ibis.expr.datatypes as dt
 import ibis.expr.operations as ops
 import ibis.expr.schema as sch
 import ibis.expr.types as ir
 from ibis import util
 from ibis.backends.base import CanCreateSchema
-from ibis.backends.base.sql.alchemy import AlchemyCrossSchemaBackend
-from ibis.backends.base.sql.alchemy.geospatial import geospatial_supported
-from ibis.backends.base.sqlglot import C, F
-from ibis.backends.duckdb.compiler import DuckDBSQLCompiler
-from ibis.backends.duckdb.datatypes import DuckDBType
+
+# from ibis.backends.base.sql.alchemy.geospatial import geospatial_supported
```

Comment:
This is a remnant of a rebase, should be removed.

Reply:
Thanks, done!